_opcodes/op058-bgee.html

```diff
@@ -24,9 +24,24 @@
 	<li>
 		Second word = Magic Weapon Dispel Type
 		<ul>
-			<li>0 &longrightarrow; Always dispel</li>
-			<li>1 &longrightarrow; Never Dispel</li>
-			<li>2 &longrightarrow; Use Caster Level</li>
+			<li>
+				0 &longrightarrow; Always dispel
+				<ul>
+					<li>Will not remove <a href="#op111">magically created</a> items flagged as <code><a href="../file_formats/ie_formats/itm_v1.htm#Header_Flags_BIT24">undispellable</a></code>.</li>
```

Comment:
Do you mean weapons in the magic slot?

Reply:
> Do you mean weapons in the magic slot?

Yes, those created via op111 (op143 is not reliable since it does not seem to equip the specified weapon...)

Reply:
Just saying that "magically created" is confusing, since they're items like any other. And not eg. the artifacts from Cespenar.